virt/post_installation_configuration/virt-post-install-storage-config.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="virt-post-install-storage-config"]
+= Post-installation storage configuration
+include::_attributes/common-attributes.adoc[]
+:context: virt-post-install-storage-config
+
+toc::[]
+
+After installing {VirtProductName}, you can configure a default storage class for the Container Storage Interface (CSI) or local storage by using the Hostpath Provisioner (HPP).
```

Comment:
awkward. The default storage class is for the OpenShift cluster, not CSI.

Reply:
I also think this should be phrased as a requirement -- "you must configure a default storage class for your cluster".